Sources/Command/Run/CommandContext.swift

```diff
@@ -71,6 +71,10 @@ public struct CommandContext {
 
         for opt in runnable.options {
             parsedOptions[opt.name] = try input.parse(option: opt)
+            
+            if case let CommandOptionType.value(defaultValue) = opt.type, defaultValue != nil  {
+                parsedOptions[opt.name] = defaultValue
```

Comment:
Will this overwrite the parsed value with the default value if a default value is found?

Reply:
Thanks for catching this. I provided test for this, since there was none.